cpp/mrc/include/mrc/modules/sample_modules.hpp

```diff
@@ -189,7 +189,7 @@ void TemplateModule<OutputTypeT>::initialize(segment::Builder& builder)
 template <typename OutputTypeT>
 std::string TemplateModule<OutputTypeT>::module_type_name() const
 {
-    return std::string(::mrc::type_name<type_t>());
+    return std::string(::mrc::boost_type_name<type_t>());
```

Comment:
Why the change from `mrc::type_name<type_t>` to `mrc::boost_type_name<type_t>`?

Reply:
mrc::type_name was a hand rolled approach; when I realized boost could do the same thing it felt reasonable to standardize on it as its likely to be more future proof.